describe.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  describePrivate,
+  PrivateDescription,
+} from "./describe/describePrivate.js";
+
+export type Description = PrivateDescription;
+
+const describeSCLElements: Record<string, (element: Element) => Description> = {
+  Private: describePrivate,
+};
+
+export function describe(element: Element): Description | undefined {
+  const describeElement = describeSCLElements[element.tagName];
```

Comment:
I think this may be written more succinctly as

```suggestion
  return describeSCLElements[element.tagName]?.(element)
```

Reply:
True thank you